app/Types/IssueType.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Types;
+
+/**
+ * The GitHub Issue Type
+ */
+enum IssueType
+{
+    /**
+     * A classic issue
+     */
+    case Issue;
+
+    /**
+     * A discussion thread
+     */
+    case Discussion;
```

Comment:
Might add complexity, needing to support the resolvement states of a third type that is relatively obscure compared to the others.

Reply:
Discussions are not supported by the REST API. See https://github.com/orgs/community/discussions/23483